consensus/src/consensus.rs

```diff
@@ -6,6 +6,7 @@ use blst::min_pk::PublicKey;
 use chrono::Duration;
 use eyre::eyre;
 use eyre::Result;
+use log::warn;
```

Comment:
Is this import being used?

Reply:
Nope, used it in testing to get timestamp vals and forgot to remove.